src/library/crate/cratestorage.cpp

```diff
@@ -453,6 +453,23 @@ CrateTrackSelectResult CrateStorage::selectTrackCratesSorted(TrackId trackId) co
     }
 }
 
+CrateTrackSelectResult CrateStorage::selectTracksSortedByCrateNameLike(const QString& crateNameLike) const {
```

Comment:
Yes, I got confused cause the other function was static and had no access to the `m_database` of CrateStorage. Should be ok now.